test/unittests/exportable_fixture.cpp

```diff
@@ -0,0 +1,47 @@
+// evmone: Fast Ethereum Virtual Machine implementation
+// Copyright 2024 The evmone Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+#ifdef _MSC_VER
+// Disable warning C4996: 'getenv': This function or variable may be unsafe.
+#define _CRT_SECURE_NO_WARNINGS
+#endif
+
+#include "exportable_fixture.hpp"
+#include <filesystem>
+#include <fstream>
+#include <regex>
+
+namespace fs = std::filesystem;
+
+namespace evmone::test
+{
+namespace
+{
+/// Creates the file path for the exported test based on its name.
+fs::path get_export_test_path(const testing::TestInfo& test_info, std::string_view export_dir)
+{
+    const std::string suite_name{test_info.test_suite_name()};
+
+    const auto stem = fs::path{test_info.file()}.stem().string();
+    const std::regex re{suite_name + "_(.*)_test"};
```

Comment:
This does not work for `eof_validation` suite in `eof_stack_validation_test.cpp` file. But I guess it makes sense to rename the file.

Reply:
Yes, let's rename it.

Reply:
Can you rebase and rename in this PR?